hraven-core/src/main/java/com/twitter/hraven/rest/RestJSONResource.java

```diff
@@ -135,18 +135,30 @@ protected SerializationContext initialValue() {
   @Path("job/{cluster}/{jobId}")
   @Produces(MediaType.APPLICATION_JSON)
   public JobDetails getJobById(@PathParam("cluster") String cluster,
-                               @PathParam("jobId") String jobId) throws IOException {
+                               @PathParam("jobId") String jobId,
+                               @QueryParam("include") List<String> includeFields)
+                                   throws IOException {
     LOG.info("Fetching JobDetails for jobId=" + jobId);
     Stopwatch timer = new Stopwatch().start();
+    Predicate<String> includeFilter = null;
+    if (includeFields != null && !includeFields.isEmpty()) {
+      includeFilter = new SerializationContext.FieldNameFilter(includeFields);
+    }
     serializationContext.set(new SerializationContext(
-        SerializationContext.DetailLevel.EVERYTHING));
+        SerializationContext.DetailLevel.EVERYTHING, null, null, includeFilter, null));
     JobDetails jobDetails = getJobHistoryService().getJobByJobID(cluster, jobId);
     timer.stop();
+    StringBuilder builderIncludeJobRegex = new StringBuilder();
+    for(String s : includeFields) {
+      builderIncludeJobRegex.append(s);
```

Comment:
How does this work? Do we just concatenate all the fields in this param without any delimiters?